packages/extension/src/components/pages/SubmitRawTransaction/SubmitRawTransaction.tsx

```diff
@@ -156,18 +156,32 @@ export const SubmitRawTransaction: FC = () => {
         <Button variant="contained" size="large" onClick={handleBack}>
           Cancel
         </Button>
-        <Button variant="outlined" size="large" onClick={handleSignTransaction} color="primary">
-          Sign
-        </Button>
-        <Button
-          variant="outlined"
-          size="large"
-          onClick={handleSubmitTransaction}
-          disabled={hasOfflineBanner}
-          color="primary"
+        <Tooltip
+          title={
+            !hasOfflineBanner
+              ? 'Note: You can directly Sign & Submit a raw transaction with the Submit button'
```

Comment:
I don't think it makes sense to talk about the submit button here as it is visible anyways.

Reply:
People get confused otherwise and think they need to sign then submi